src/sections/Company/Cloud-Native/cloud-native.style.js

```diff
@@ -57,11 +57,11 @@ const CloudNativeWrapper = styled.div`
 
     }
     .btn-cont button{
-        color:${props => props.theme.DarkTheme ? "black" : "white"};
+        color:black;
     }
     .btn-cont button:hover{
-        color:${props => props.theme.DarkTheme ? "white" : "black"};
-        box-shadow:0 2px 10px rgba(0,0,0,0.4);
+        color:white;
+        box-shadow: rgb(255 255 255 / 40%) 0px 2px 10px;
```

Comment:
The proposal is to use the same color for both light and dark modes, is that right?

Reply:
yes @leecalcote , this change is corresponding to the text inside the yellow "About Us" button on the home page. This text inside is currently white on the default (light-mode) page and black on (dark-mode) page.  To pass the accessibility test, I have set it to black on both the pages. And since the text is black on both modes ,I have set the hover color to white on both.
If we want, we can leave the things as they were for the dark mode as the lighthouse-ci evaluates the page on the default mode only.